Changes the autopsy scanner icon, attack self calls print_data

Miraviel opened the pull request:
:cl:
tweak: Using the autopsy scanner in-hand will now print the scan. Its icon was also updated.
/:cl:

Printing data was done by right-click -> print data, I added it to attack_self for making it easier.
A bit prone to spam, if people spam scans, I'll add a delay to it.

I hope the new icon is satisfactory, I used the morgue tray's colour palette to make it fitting.

Preview:
![scanner](https://i.imgur.com/IvKeHP9.gif)

Miraviel committed:
Changes the autopsy scanner icon; attack self calls print_data

Roland410 commented:
Probably should get rid of the menu prompt as it's redundant now.

afterthought2 commented on code/game/objects/items/weapons/autopsy.dm:
```diff
@@ -187,3 +187,7 @@
 
 	return 1
+
+/obj/item/weapon/autopsy_scanner/attack_self(mob/user as mob)
```
Get rid of `as mob`; it's verb syntax that does nothing here.

afterthought2 commented on code/game/objects/items/weapons/autopsy.dm:
```diff
@@ -187,3 +187,7 @@
+
+/obj/item/weapon/autopsy_scanner/attack_self(mob/user as mob)
+	print_data(user)
+	return 0
```
Don't need to explicitly return `0` here; `attack_self`'s returns aren't checked.

Miraviel replied:
Will do once I get home. Should it be the same with [the plant analyzer](https://github.com/Baystation12/Baystation12/blob/a4ecca2d2c7368b37b2529cda6abd131d5cbd3dc/code/modules/hydroponics/trays/tray_tools.dm#L43)? I used it as an example as I was not sure.

afterthought2 replied:
Yup. The `as mob` thing was common in old code; don't know if returns on this proc were ever meaningful, but they aren't anymore.

afterthought2 commented:
@Miraviel, while you are at it, could you move the icons to the dmi with the other surgery equipment, and give the icon_state a name? The convention it uses currently is kind of strange and not consistent with other stuff.

Miraviel committed:
Removed superfluous code, moved icon

Miraviel commented:
Removed `as mob` and `return 0` from both the autopsy scanner and the plant analyzer.
Moved the new and old icons to `surgery.dmi` and gave the the names `autopsy_scanner` and `autopsy_scanner_old`.
Deleted `autopsy_scanner.dmi`.

Tested, they work perfectly!
